GVFS/GVFS.Service/GVFSMountProcess.Mac.cs

```diff
@@ -0,0 +1,47 @@
+﻿using GVFS.Common;
+using GVFS.Common.Tracing;
+using GVFS.Service.Handlers;
+using System;
+
+namespace GVFS.Service
+{
+    public class GVFSMountProcess : IDisposable
+    {
+        private const string ParamPrefix = "--";
+
+        private readonly ITracer tracer;
+
+        public GVFSMountProcess(ITracer tracer, int sessionId)
+        {
+            this.tracer = tracer;
+        }
+
+        public bool Mount(string repoRoot)

```

Comment:
A lot of the code in `Mount` appears to be identical to the code in `GVFSMountProcess.Windows.cs`, can we share it?

Reply:
Agree. I've restructured this class for UT, can we defer this review for that PR.